chains/celo/listener_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package celo
+
+import (
+	"testing"
+)
+
+func Test_Start(t *testing.T) {
+	l := &listener{}
+	err := start(l)
```

Comment:
is `conn.Connection()` defined for this listener? (see line 18 of `celo/listener.go`)

Reply:
I see, because `l.conn` is an interface do I need to define/create a `Connection` method in this testfile?